### DISCUSSION TITLE
Needed Help

### DISCUSSION THREAD

**Discussion by Riteshag05** [plausible/analytics#4608] [Self-Hosted Support]
I have added the script in my frontend code but still plausible instance is not showing anything.

**Comment by ruslandoga** [comment]
👋 @Riteshag05 

You would need to provide more information. Right now it's impossible for us to know what is going wrong :)

**Reply by ruslandoga** [reply]
Please list the exact steps you performed to install Plausible, all your configuration files, and anything else relevant.

**Reply by Riteshag05** [reply]
Can we connect if possible at 12:40
https://meet.google.com/hpf-tmip-tsy
this is the link, i am waiting i won't be able to explain here

**Reply by ruslandoga** [reply]
No, it's already 14:10 where I am :)

**Reply by Riteshag05** [reply]
Can we connect now?

**Reply by ruslandoga** [reply]
Customer support is not available in CE. You would need to purchase one of the paid plans for someone to "connect" with you: https://plausible.io/#pricing

**Reply by Riteshag05** [reply]
(1) i cloned this "https://github.com/plausible/community-edition"
(2) changed the BASE_URL, SECRET_KEY_BASE, TOTP_VAULT_KEY
(3) In BASE-URL = http://localhost:8000/, i added this
(4) and for other SECRET AND TOTP , generated their value
(5) after doing this Plausible instance comes up

and also installed the docker

**Reply by Riteshag05** [reply]
Added this snippet in my frontend code which is written in Angular
<script async defer data-api="http://localhost:8000/api/event" data-domain="localhost" src="http://localhost:8000/js/script.js"></script>

**Reply by Riteshag05** [reply]
<img width="853" alt="Screenshot 2024-09-24 at 12 31 07 PM" src="https://github.com/user-attachments/assets/e37e4185-5dfd-4378-a486-18f0c229db74">

now plausible is setup and i am logged in to my frontend url but still the count is 0.

**Reply by ruslandoga** [reply]
Why do you use localhost?

**Reply by ruslandoga** [reply]
Is your Angular app local too?

**Reply by Riteshag05** [reply]
Yes angular is running locally

**Reply by ruslandoga** [reply]
I think Plausible is blocking local requests by default.

**Reply by Riteshag05** [reply]
so anything that you can suggest

**Reply by ruslandoga** [reply]
You can deploy both your Angular app and Plausible on the internet, I guess.
